tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-dashboard.html

```diff
@@ -465,12 +466,14 @@ <h2>Continue...</h2>
 
         _leftMenuWidth: {
           type: Number,
-          value: 450,
+          value: tf_storage.getNumberInitializer(
+              '_leftMenuWidth', {defaultValue: _DEFAULT_LEFT_MENU_WIDTH}),
```

Comment:
Another nit: this is not a "menu". A more appropriate word would be "pane" or "region". WDYT?

Reply:
+1, renamed in https://github.com/tensorflow/tensorboard/pull/856